.ci/docker/common/install_docs_reqs.sh

```diff
@@ -7,7 +7,13 @@ if [ -n "$KATEX" ]; then
   # Ignore error if gpg-agent doesn't exist (for Ubuntu 16.04)
   apt-get install -y gpg-agent || :
 
-  curl --retry 3 -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -
+  # The original script downloaded https://deb.nodesource.com/setup_16.x and
+  # immediately piped it into bash, which breaks security-checking software;
+  # the solution was to download the file and store it in the current
+  # directory -- this requires (for checkin) that a human read the file
+  # to make sure it's not doing anything naughty, and if it ever needs
+  # updating, that is easily done with a download and pull request
+  cat setup_16.x | sudo -E bash -
```

Comment:
This part of the code is ancient, I think we should try to get rid of it and see if we can use nodejs from apt-get

Reply:
Hi -- Can we push this change now and readdress this later?  I'm getting nagged by my security folks, and it's going to take me a while to figure out how to do the nodejs solution... :-)
Thanks!
Tim